_layouts/docs.html

```diff
@@ -53,6 +53,10 @@ <h3 class="toc-title{% if isCurrentSection %} active{% endif %}">{{ section.titl
                 <h1 class="page-header">{{ page.title }}</h1>
                 <div class="edit-panel" style="margin: 0 0 1em 0;">
                     <a href="{{ site.slim_website_repo }}{{page.path}}" target="_blank" class="btn btn-default btn-xs"><i class="fa fa-github"></i> Edit This Page</a>
+                    <select onchange="this.options[this.selectedIndex].value && (window.location = this.options[this.selectedIndex].value);">
+                        <option value="/docs/v2">v2</option>
+                        <option value="/docs/" selected>v3</option>
```

Comment:
I think the value of this should be `/docs/v3` rather than just `/docs/`. I would rather not rely on the redirect for this.

Reply:
The trick here is that `docs/v3` doesn't have an index file since it's the current version of the docs. The thought would be once Slim 4 is the current version, this would then be changed to `/docs/v3` and the current file at `docs/index.md` would be moved to `docs/v3/index.md`

Reply:
This makes sense.

Reply:
Actually wait, won't all urls have the doc version in it? even the current version of docs?

Reply:
Right, the only docs page that *won't* have the version would be the index of the current version. When we roll over to a new version, we'd shift then.

Alternatively we could move `docs/index.md` -> `docs/v3/index.md` but we'd need to create a new `docs/index.md` but that seems like extra or duplicate content.

Reply:
Ah now I see what you mean. We are good to merge then.